client/components/swipeable/style.scss

```diff
@@ -0,0 +1,29 @@
+@import '@wordpress/base-styles/mixins';
+.swipeable__container {
+	width: 100%;
+	height: 100%;
+	position: relative;
+	overflow: hidden;
+	list-style: none;
+	padding: 0;
+	touch-action: pan-y;
+}
+
+.swipeable__pages {
+	position: relative;
+	width: 100%;
+	height: 100%;
+	z-index: 1;
+	display: flex;
+
+	transition: all 0.5s ease-in-out;
+	transition-property: transform, height;
+	@include reduce-motion( 'transition' );
+}
+.swipeable__resize-observer {} /* stylelint-disable-line block-no-empty */
```

Comment:
Should we remove this?

Reply:
For some reason I thought it was needed but after testing again. it doesn't seem to be the case anymore. 🙃